Mumble does not compile with `-Dmanual-plugin=OFF`

**Describe the bug**
Subj.

**Steps to Reproduce**
1. Try to build `master` with `-Dmanual-plugin=OFF -Dserver=OFF -Doverlay=OFF`
2. Observe the error:
    ```sh
    [ 87%] Building CXX object src/mumble/CMakeFiles/mumble.dir/Plugin.cpp.o
    [ 87%] Building CXX object src/mumble/CMakeFiles/mumble.dir/PluginInstaller.cpp.o
    In file included from /home/user/code/mumble/src/mumble/PluginConfig.cpp:10:
    /home/user/code/mumble/src/mumble/ManualPlugin.h:15:10: fatal error: ui_ManualPlugin.h: No such file or directory
       15 | #include "ui_ManualPlugin.h"
          |          ^~~~~~~~~~~~~~~~~~~
    compilation terminated.
    gmake[2]: *** [src/mumble/CMakeFiles/mumble.dir/build.make:1199: src/mumble/CMakeFiles/mumble.dir/PluginConfig.cpp.o] Error 1
    gmake[2]: *** Waiting for unfinished jobs....
    gmake[1]: *** [CMakeFiles/Makefile2:1454: src/mumble/CMakeFiles/mumble.dir/all] Error 2
    gmake: *** [Makefile:171: all] Error 2
    ```

**Expected behavior**
Mumble compiles.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Debian 11
 - Version: `master`

**Additional context**
N/A

--- a/src/mumble/PluginConfig.cpp
+++ b/src/mumble/PluginConfig.cpp
@@ -7,7 +7,6 @@
 
 #include "Log.h"
 #include "MainWindow.h"
-#include "ManualPlugin.h"
 #include "Message.h"
 #include "MumbleApplication.h"
 #include "PluginInstaller.h"

--- a/src/mumble/PluginManager.cpp
+++ b/src/mumble/PluginManager.cpp
@@ -22,12 +22,15 @@
 
 #include "API.h"
 #include "Log.h"
-#include "ManualPlugin.h"
 #include "PluginInstaller.h"
 #include "PluginUpdater.h"
 #include "ProcessResolver.h"
 #include "ServerHandler.h"
 #include "Global.h"
+
+#ifdef USE_MANUAL_PLUGIN
+#	include "ManualPlugin.h"
+#endif
 
 #include <cstdint>
 #include <memory>

--- a/src/mumble/ServerHandler.cpp
+++ b/src/mumble/ServerHandler.cpp
@@ -31,6 +31,7 @@
 #include "Utils.h"
 #include "Global.h"
 
+#include <QPainter>
 #include <QtCore/QtEndian>
 #include <QtGui/QImageReader>
 #include <QtNetwork/QSslConfiguration>

